docs/Release_notes.md

```diff
@@ -1,6 +1,199 @@
+-	Issue 39 / PR 40: Added a configurable start delay for the carma-platform's system to be fully ready - route is selected and semantic map is available in all plugins/nodes. Without it, the carma-platform shuts down automatically almost half of the time.
+-	PR 41: Added detection node that leverages carla-sensor-lib to publish object level detection data, replacing the current external objects node
+-	PR 44: Ackermann controller was throwing a KeyError when carma-base image version was changed to develop from 4.2.0. To fix this issue, the install.sh was modified to include specific version 1.0.1 of "simple-pid" package and the carma_carla_agent.launch was updated to include the newly added start delay parameter
+-	PR 58: Adjusted config to use map centric instead of sensor centric object detection
+-	PR 67: Added script to wait for CARLA. This script can replace sleep commands that were environment specific
+-	PR 68: Added workflow dispatch as carma-carla-integration tool depends on libraries in carla-sensor-lib. It should trigger rebuild if carla-sensorl-lib repo has new changes pushed.
+
+Fixes in this release:
+-	PR 64: Fixed error log statements caused by incorrect syntax in launch file.
+
+### **CARMA NS3 Adapter** 
+
+Enhancement in the release:
+-	PR 11: Added udp listener to listen to time sync messages from CDASim and broadcast it in rostopic as /sim_clock.
+-	PR 13: Added wave JSON configurations for SDSM.
+
+### **CARLA Sensor Library** 
+
+The carla-sensor-lib is a new repository and docker image for this release. This library provides functions to create CARLA sensors with noise modeling and retrieve detection information from created sensors. Additionally this repository creates a docker image which deploys and XML RPC server that allows clients to create sensor in CARLA and poll their detections. This new feature is provided to enable object detection for CDASim deployment using CARLA's existing sensor and object representation. The data will be feed to the CARMA-Platform via the CARLA CARMA Integration tool, and to CDASim to expose the data to other integrated simulators or software systems under test.
+
+### **CARLA ScenarioRunner** 
+
+The scenario-runner is a new repository and docker image for this release. This directory contains custom ScenarioRunner scenario configurations to facilitate integration testing. This still a work in progress, so the scenarios serve more as example references. ScenarioRunner's code resides in the srunner Python package, and the scenario_runner.py script is responsible for launching scenarios. 
+
+### **EVC-SUMO (New Private Repository)** 
+
+This EVC-sumo is a new private repository. Econolite Virtual Controller (EVC)-SUMO bridge tool that is a part of XiL co-simulation tool. This bridge acts as a mediator between EVC (via the PyEOS Python package) and SUMO, integrating the exchange of traffic light status and detector status.
```

Comment:
EVC-sumo is a new private repository featuring the Econolite Virtual Controller (EVC)-SUMO bridge tool, part of the XiL co-simulation toolset. This bridge acts as a mediator between EVC, accessed via the PyEOS Python package, and SUMO, facilitating the integration of traffic light and detector status exchanges. 

Currently, building and using the Docker image for this tool requires a proprietary GitHub token, accessible only to members of the USDOT-FHWA-STOL organization, due to licensing restrictions imposed by Econolite.

Reply:
updated